contents/blog/clickhouse-vs-snowflake.md

```diff
@@ -21,105 +21,111 @@ If I was to compare the databases to boats, ClickHouse is a jet ski – ultra-fa
+Separately, an advantage of ClickHouse Cloud is that it includes some exclusive features not available in the ClickHouse core distribution. One of these is a SQL GUI explorer which provides similar features to some of Snowflake’s basic data exploration. 
+
+Overall, ClickHouse Cloud or Altinity Cloud are excellent options for companies that want a managed version of ClickHouse on the public cloud. Their emergence closes the gap between ClickHouse and Snowflake. 
 
 ## Architecture overview
 
 When Snowflake was released in 2012, there were two major paradigms – **Shared-Disk Architecture** and **Shared-Nothing Architecture**.
 
 ![Shared Disk Architecture.png](../images/blog/clickhouse-vs-snowflake/shared-disk-architecture.png)
 
-Under Shared-Disk Architecture, CPU and memory were split into nodes, but each connected to a single storage unit (these days usually a blob store like S3 or GCS). The benefit of Shared-Disk Architecture is that you can scale Disk and Compute separately. This is huge when you are thinking about a data warehouse when your workloads my be very spiky and periodic but your data has to be stored all the time. The cost here is that typically reading from disk is a bit slower, at least in terms of latency, when compared to reading from local storage. 
+Under Shared-Disk Architecture, CPU and memory were split into nodes, but each connected to a single storage unit. The benefit of Shared-Disk Architecture is that data is always in sync, but collective usage means stalled requests to avoid conflicts.
 
 ![Shared Nothing Architecture.png](../images/blog/clickhouse-vs-snowflake/shared-nothing-architecture.png)
 
-Meanwhile, CPU, memory, and storage are encapsulated under Shared-Nothing Architecture in separate, parallel instances, only syncing via background jobs. The benefit of Shared-Nothing Architecture is speed. There is a risk that storage could go out of sync since usually replication here is eventually consistent, but that was the case with blob stores up until recently too. The easiest way to think about this is consider Postgres. When you install Postgres everything is boxed up nice and neat on a single instance. At smaller scale this is a much simpler setup and enables you to run the architecture pretty much anywhere.
+Meanwhile, CPU, memory, and storage are encapsulated under Shared-Nothing Architecture in separate, parallel instances, only syncing via background jobs. The benefit of Shared-Nothing Architecture is an arbitrary instance never stalls requests; however, storage could go out of sync. 
```

Comment:
```suggestion
Meanwhile, CPU, memory, and storage are encapsulated under Shared-Nothing Architecture in separate, parallel instances, only syncing via background jobs. The benefit of Shared-Nothing Architecture is speed. There is a risk that storage could go out of sync since usually replication here is eventually consistent, but that was the case with blob stores up until recently too. The easiest way to think about this is consider Postgres. When you install Postgres everything is boxed up nice and neat on a single instance. At smaller scale this is a much simpler setup and enables you to run the architecture pretty much anywhere.
```